<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>757</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>891</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000000</width>
    <height>5000000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="TabWidgetMain">
      <property name="minimumSize">
       <size>
        <width>870</width>
        <height>410</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="GBOX_bottom">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="NON_detector">
         <property name="text">
          <string>Detector</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LBL_detector">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="BTN_connect_sequencer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Connect FPGA</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="NON_X_pos">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LBL_X_pos">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="NON_Y_pos">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LBL_Y_pos">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="NON_Z_pos">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LBL_Z_pos">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BTN_MOVE_pos">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>MOVE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BTN_READ_pos">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>READ</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BTN_MOVE_pos</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pressedMovePosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>864</x>
     <y>718</y>
    </hint>
    <hint type="destinationlabel">
     <x>837</x>
     <y>895</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_READ_pos</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pressedReadPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>950</x>
     <y>718</y>
    </hint>
    <hint type="destinationlabel">
     <x>1072</x>
     <y>895</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_connect_sequencer</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>pressedConnectSequencer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>718</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>860</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LBL_detector</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>returnedDetector()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>705</y>
    </hint>
    <hint type="destinationlabel">
     <x>-104</x>
     <y>725</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pressedConnectSequencer()</slot>
  <slot>pressedMovePosition()</slot>
  <slot>pressedReadPosition()</slot>
  <slot>pressedMotorSettings()</slot>
  <slot>returnedDetector()</slot>
 </slots>
</ui>
